www/index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
```

Comment:
The code is very clean and the use of the functions from your .js files being called within the index file makes it enjoyable to read through without getting lost or stuck.